app/shared/src/commonMain/kotlin/ui/settings/tabs/app/AppSettingsTab.kt

```diff
@@ -440,6 +440,17 @@ private fun SettingsScope.PlayerGroup(
             title = { Text("自动连播") },
             Modifier.placeholder(vm.uiSettings.loading),
         )
+        if (currentPlatform.isDesktop()) {
+            HorizontalDividerItem()
+            SwitchItem(
+                checked = config.skipOped,
+                onCheckedChange = {
+                    vm.videoScaffoldConfig.update(config.copy(skipOped = it))
+                },
+                title = { Text("跳过oped") },
```

Comment:
```suggestion
                title = { Text("自动跳过 OP 和 ED") },
```
- 中英文之间有空格
- OP 是缩写, 应当大写
- OP/ED 是两个词不应该写到一起